test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -727,8 +727,10 @@ test.describe( 'Image', () => {
 		expect( src ).toMatch( /\/wp-content\/uploads\// );
 	} );
 } );
-
-test.describe( 'Image - interactivity', () => {
+// These tests are disabled for now while we
+// work on a solution to make them more stable.
+// See https://github.com/WordPress/gutenberg/pull/52442
+test.skip( 'Image - interactivity', () => {
```

Comment:
```suggestion
test.describe.skip( 'Image - interactivity', () => {
```

This should be `test.describe.skip` since it's a group of tests that we want to skip - https://playwright.dev/docs/api/class-test#test-describe-skip

Reply:
@Mamaduka I gave that a shot and it didn't work! Not sure what the issue may be.

Reply:
@Mamaduka Would you be able to give the `test.describe.skip` syntax a try on your machine to see if it performs as expected?

Reply:
It works in my tests.

```
npm run test:e2e:playwright -- test/e2e/specs/editor/blocks/image.spec.js
```

## Screenshot
![CleanShot 2023-07-27 at 16 54 54](https://github.com/WordPress/gutenberg/assets/240569/9cb0d138-8429-4462-9495-fc8be366c787)

Reply:
Got it! Turns out it does in mine as well and I was misreading the logs. Changes pushed.